tinker/e_announcements/__init__.py

```diff
@@ -125,10 +125,10 @@ def duplicate(self, e_announcement_id):
 
     @route("/submit", methods=['post'])
     def submit(self):
-        rform = request.form
+        rform = EncodingDict(request.form)
         eaid = rform.get('e_announcement_id')
 
-        form, passed = self.base.validate_form(rform)
+        form, passed = self.base.validate_form(rform.internal_dictionary())
```

Comment:
WTForm's built-in validator takes in an ImmutableMultiDict in order to build the form with data in it already, so I had to pass in the `internal_dictionary` to the validator. I did the same for Events and Fac Bios. I figured that would be ok since the validators are designed to work with ImmutableMultiDicts which store all strings as unicode anyway, and everywhere else that uses EncodingDicts makes .get calls which encode it.